docs/runbooks/deploy-to-library-site.md

```diff
@@ -0,0 +1,80 @@
+# Deploy new version to library site(s)
+
+## When to use
+
+If you want to deploy a new version of `dpl-cms` to [one or more library sites
+which have been added to the platform](add-library-site-to-platform.md).
+
+## Prerequisites
+
+* A local checkout of the [`dpl-platform` repository](https://github.com/danskernesdigitalebibliotek/dpl-platform)
+* A running [dplsh](using-dplsh.md) with `DPLPLAT_ENV` set to the platform
+  environment name.
+* The name of the version you want to deploy. This must correspond to a
+  tagged version of the [`dpl-cms-source` image](https://github.com/danskernesdigitalebibliotek/dpl-cms/pkgs/container/dpl-cms-source).
+
+### Procedure: New version for testing
+
+Use this procedure to deploy a new version for testing before it is released to
+production library sites.
+
+1. In your local environment ensure that your checkout of the `main` branch for
+   `dpl-platform` is up to date.
+2. Create a new branch from `main`.
+3. Update `infrastructure/environments/dplplat01/sites.yaml` by setting the
+   value for `dpl-cms-release` for the site `canary` to the new version to use
+   for the site to update.
+4. Commit the change and push your branch to GitHub and create a pull request.
+5. Request a review for the change and wait for approval.
+6. Start `dplsh` from the `/infrastructure` directory of your local
+   environment.
```

Comment:
[Some documentation refers to `dplsh` being symlinked into your path](https://github.com/danskernesdigitalebibliotek/dpl-platform/blob/ca9d7777b0873715156add02b21019ed49f1b5db/tools/dplsh/README.md#launching-the-shell). This would make the path part obsolete. I have not done so myself though.

Does this change your suggestion?